.github/workflows/test-tooling.yml

```diff
@@ -0,0 +1,59 @@
+name: Test Tooling
+
+on:
+  push:
+    paths: # Explicitly declare which paths
+      - ".github/workflows/test-tooling.yml"
+      - "tools/*"
+  pull_request:
+    branches:
+      - master
+    paths: # Explicitly declare which paths
+      - ".github/workflows/test-tooling.yml"
+      - "tools/*"
+
+jobs:
+  lint:
+    name: Test (${{ matrix.os}}) go ${{ matrix.gover }}
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        gover:
+          - "1.21"
+        os: 
```

Comment:
I think it should run on as many OSes as possible with this workflow already catching that LF-style endings were not applied on Windows - especially since this workflow is not resource intensive and only blocks ~1min of action runner time.

Reply:
I think it should run on as many OSes as possible with this workflow already catching that LF-style endings were not applied on Windows - especially since this workflow is not resource intensive and only blocks ~1min of action runner time. I believe this should be kept but have no overly strong feelings on it either way; I'm happy to remove if this is the consensus reached.

Reply:
This only runs if you change `.build-tools` and `.github/workflows/test-tooling.yml`, right?

Reply:
Ok i think we can live with 3 OS's then IMHO